ethcore/blockchain/src/generator.rs

```diff
@@ -136,6 +138,29 @@ impl BlockBuilder {
 		})
 	}
 
+	/// Add a block with randomly generated transactions.
+	#[inline]
```

Comment:
This is a test helper yes? Is it performance sensitive?

Reply:
Yeah I think it's only used for tests. I just followed the structure of the other similar methods, so I'm not sure whether `inline` is needed here.